udata/harvest/backends/base.py

```diff
@@ -84,11 +84,8 @@ def perform_initialization(self):
             if not self.dryrun:
                 self.job.save()
         except Exception as e:
-            log.error("Error in initialization : %s" % (str(e)))
-            log.exception(e)
             self.job.status = 'failed'
-            error = HarvestError(message=str(e),
-                                 details=traceback.format_exc())
```

Comment:
Not a problem because as stated in the description:
> (no relevant to the end user and already available while developping in the console or in Sentry in production)

This only remove the stacktrace in the admin interface